src/sqlancer/tidb/TiDBProvider.java

```diff
@@ -84,6 +87,7 @@ private static int mapActions(TiDBGlobalState globalState, Action a) {
         case CREATE_INDEX:
             return r.getInteger(0, 2);
         case INSERT:
+            return r.getInteger(0, globalState.getOptions().getMaxNumberInserts());
```

Comment:
Interesting, I didn't notice that we previously generated only up to two `INSERT` statements.